sims/verilator/Makefile

```diff
@@ -96,11 +97,12 @@ $(model_mk_debug): $(sim_vsrcs) $(sim_common_files) $(INSTALLED_VERILATOR)
 #########################################################################################
 # invoke make to make verilator sim rules
 #########################################################################################
+VERILATOR_MAKEFLAGS=-j8
```

Comment:
Can we share this flag with the Verilator build step and `make -j*` the Verilator binary... since that also takes a long time in CI.

Reply:
No, because verilator compile is just a few minutes, and this flag exists only to deal with the fact that `make -jN` at the top-level makefile is broken (Issue #147)

Reply:
10mins is a long time!

Reply:
Isn't the installed verilator cached between runs anyways?

Reply:
Yes it is. Its really not an issue.